contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
shouldn't this file be in the operations folder ?

Reply:
it is. this is an old version that I have now removed.